github/messages.go

```diff
@@ -56,6 +56,7 @@ var (
 		"page_build":                            "PageBuildEvent",
 		"public":                                "PublicEvent",
 		"pull_request_review_comment":           "PullRequestReviewCommentEvent",
+		"pull_request_review":                   "PullRequestReviewEvent",
```

Comment:
I suppose so. That seems to be the order in other files (activity_events.go, event_types.go, messages_test.go, etc.), and it's alphabetical ordering, right?